pkg/util/test/util.go

```diff
@@ -3555,15 +3555,21 @@ func validateTelemetryStatusInPxctl(telemetryShouldBeEnabled bool, cluster *core
 				return nil, true, fmt.Errorf("got error while trying to get Telemetry status from pxctl, Err: %v", err)
 			}
 
-			if telemetryShouldBeEnabled && !strings.Contains(output, "Healthy") {
-				return nil, true, fmt.Errorf("[%s (%s)] Telemetry is enabled and should be Healthy in pxctl status on PX node, but got [%s]", pxPod.Spec.NodeName, pxPod.Name, strings.TrimSpace(output))
-			} else if !telemetryShouldBeEnabled && !strings.Contains(output, "Disabled") {
-				return nil, true, fmt.Errorf("[%s (%s)] Telemetry is not enabled and should be Disabled in pxctl status on PX node, but got [%s]", pxPod.Spec.NodeName, pxPod.Name, strings.TrimSpace(output))
-			} else if !strings.Contains(output, "Disabled") && !strings.Contains(output, "Healthy") {
-				return nil, true, fmt.Errorf("[%s (%s)] Telemetry is Enabled=%v, but pxctl on PX node returned unexpected status [%s]", pxPod.Spec.NodeName, pxPod.Name, telemetryShouldBeEnabled, output)
+			// Find Telemetry status line
+			telemetryStatusString := regexp.MustCompile("Telemetry:.*").FindString(output)
```

Comment:
Might want to change the regex to pickup line "beginning with"...  So maybe use "^Telemetry:.*" just in case we add warning or something later on...

Reply:
Done